content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -34,21 +34,35 @@ further_reading:
     tag: "GitHub"
     text: "Source code"
 ---
+## Capabilities
```

Comment:
If we're going to add a "Capabilities" section, we should add a bullet point for the main features of the tracer (generating distributed traces) and add it at the top of the list.

Reply:
This section has been removed.

Reply:
Updated and reflected in https://github.com/DataDog/documentation/pull/11200/commits/77793e5a94b644a2679fb90ab651acd9a154876b